flake8_trio.py

```diff
@@ -11,7 +11,18 @@
 
 import ast
 import tokenize
-from typing import Any, Dict, Iterable, List, NamedTuple, Optional, Set, Union
+from typing import (
+    Any,
+    Dict,
+    Iterable,
+    List,
+    NamedTuple,
+    Optional,
+    Set,
+    Tuple,
+    Union,
+    cast,
+)
```

Comment:
damn I long for when 3.8 is EOL, a pain to have to add stuff up here all the time (should look for if there's any good neovim plugins that suggests and automatically adds imports)